src/main/java/org/broadinstitute/hellbender/tools/copynumber/FilterIntervals.java

```diff
@@ -431,6 +432,17 @@ private SimpleIntervalCollection filterIntervals() {
                     countNumberPassing(mask), numIntersectedIntervals));
         }
 
+        //finally, filter intervals that are solitary in their corresponding contigs
+        final Map<String, Long> contigToCountMap = IntStream.range(0, numIntersectedIntervals)
+                .filter(i -> !mask[i]).mapToObj(i -> intersectedIntervals.getRecords().get(i))
+                .collect(Collectors.groupingBy(SimpleInterval::getContig, Collectors.counting()));
+        IntStream.range(0, numIntersectedIntervals).filter(i -> !mask[i]).forEach(i -> {
+            final long value = contigToCountMap.get(intersectedIntervals.getRecords().get(i).getContig());
+            if (value <= 1) {
+                mask[i] = true;
```

Comment:
Perhaps a log message here indicating when a contig is dropped?

Reply:
Done!